### ISSUE TITLE
Checked checkboxes not visible on Windows High Contrast Mode (WCAG Level AAA)

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#2248]
When using Windows HCM, you can't see when a checkbox is checked.

![Screenshot 2022-06-16 at 15 23 49](https://user-images.githubusercontent.com/43346934/174091657-786bd34c-065a-431f-bb79-42d991b64725.png)

Page 70-71 of June 2022 DAC DS report...

---

## Inverted Contrast
Content is missing when switching to Windows’ High Contrast Mode.
WCAG Reference:
1.4.8 Visual Presentation (Level AAA)
Understanding Visual Presentation | How to Meet Visual Presentation
### Issue ID: DAC_Inverted_Contrast_01
URL: https://ons-design-system.netlify.app/patterns/download-resources/examples/download-resources/
Page title: Service name
Journey: Journey 2, Step 2

It may be difficult for low vision users to see which checkboxes are checked while the colours are inverted to reduce screen glare. The check within the checkboxes are not visible with colour inversion.

### Current code ref(s):
#local-authorities 
```
<input type="checkbox" id="local-authorities" class="ons-checkbox__input ons-js-checkbox" value="local-authorities" data-filter="local-authorities">
```

### Low vision comments: 
“When I was looking at the filters on the page with colour inversion enabled, I found that upon selecting the checkboxes in any of the filter options available that the checkmarks would not appear therefore not making it clear that the checkboxes have been checked and that the filter has been adjusted.”
Additional instances of this issue may exist on other pages throughout the service; wherever this issue occurs, they too will need to be resolved.

### Solution:
We advise that checkboxes are remediated to display a checked state with inverted colours. This eliminates the need of the user returning to standard colours which may induce screen glare.
